frontend/app/services/AuthenticationService.scala

```diff
@@ -20,6 +20,11 @@ trait AuthenticationService {
     user = guUCookieData.user if user.id == minimalSecureUser.id
   } yield IdMinimalUser(user.id, user.publicFields.displayName)
 
+  def identityAuthenticatedUser[A](request: RequestHeader): Option[IdMinimalUser] = for {
```

Comment:
Very curious about this. Why has this been added, when `authenticatedUserFor()` exists above, and does very similar stuff?